test/util/testutil/types.go

```diff
@@ -100,6 +100,8 @@ const (
 	Default SSLMode = ""
 	// Disable SSLMode
 	Disable = "disable"
+	// Prefer SSLMode
+	Prefer = "prefer"
```

Comment:
I think the new test cases get us to a good place, but we might have to find another place for this value. `types.go` contains values that are really dedicated to `AbsractionConfiguration` i.e. `Secretless -> Target Service`. These are values common to the Secretless DB connectors configuration. I think somewhere closer to `ClientConfiguration` would be better, though "prefer" is really just a concept in PostgeSQL (I think).

I'll see if I can suggest a better place for it.

Reply:
I removed the constant completely, it's just as a literal in the test. If you have a better idea, let me know and I'll fix it.

Reply:
Thank you. I'd like to keep `types.go` decoupled from the clients. I think for now we can use just literal strings for `ClientConfiguration.SSLMode`.

Reply:
A better way would probably be to define sslmode string constants local to each client.